frameworks/elastic/src/main/dist/setup-keystore-bootstrap-password.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+# This script sets the Elasticsearch bootstrap password so that requests to the cluster work out of the box without
+# requiring users to set passwords, while still recommending them to do so if they haven't already.
+# The bootstrap password is ignored if users already set passwords.
+
+readonly SCRIPT_NAME="$(basename "${BASH_SOURCE[0]}")"
+readonly ELASTICSEARCH_PASSWORD="changeme" # can't get it from env because it's not exposed as TASKCFG_ALL.
```

Comment:
Could we use environment secrets from the secret store?

Reply:
I suspect that as part of security we're going to wind up pulling this from the secret store. I think this is fine for now. This maintains parity with what we do currently right, @mpereira?

Reply:
@benclarkwood yes.